Directory.Build.props

```diff
@@ -29,9 +30,15 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0' or '$(TargetFramework)' == 'net7.0'">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0' or '$(TargetFramework)' == 'net7.0' or '$(TargetFramework)' == 'net8.0'">
```

Comment:
>PropertyGroup [](http://example.com/codeflow?start=3&length=13)

I wonder if we can implement some of the suggestions from the .NET team...there was a PR in wilson which was not merged, but might be a place to start.

Reply:
Apparently, using 'TargetFrameworkIdentifier' in anything that OWIN references (e.g. `TokenAcquisition.csproj`) causes the build to fail. I will implement the suggestions elsewhere, though.